pygfx/renderers/wgpu/meshshader.py

```diff
@@ -128,6 +131,35 @@ def get_bindings(self, wobject, shared):
         sampler = GfxSampler(material.map_interpolation, "repeat")
         texturing = "texture/auto"
 
+        # todo, check uv is present and has the right shape. (introduce uv channel for texture)
+
+        # specular map configs, for basic and phong materials
+        if (
+            getattr(material, "specular_map", None)
+            and type(material) is not MeshStandardMaterial
+        ):
```

Comment:
This shader is for `MeshBasicMaterial`, so I would expect that all properties  that are available on `MeshBasicMaterial` can be accessed via `material.xx`. There are a few cases where `getattr()` is still used, from before we refactored the material class hierarchy.


```suggestion
        if material.specular_map and not isinstance(material, MeshStandardMaterial):
```

Reply:
`MeshNormalShader`, `MeshSliceShader`, and other shaders inherit from this shader, but these materials do not have the "specular_map" property.

Reply:
Ah, right ... the materials don't but the *shader classes* do ...